### PR TITLE
Try to bum gdal version

### PR DESCRIPTION
**Pull request by SrNetoChan**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Trying to get gdal to bump, so we can add pdal to qgis feedstock.

### CODE REVIEW

**Comment by SrNetoChan** [comment]
@conda-forge-admin, please rerender

**Comment by SrNetoChan** [comment]
Even if this passes, I am pretty sure there's a better/correct way of bumping a dependency without messing the files created by the conda smithy. @ocefpaf can you tell me where is that done?

**Comment by hobu** [comment]
GDAL and PROJ bumps come through Conda Forge's pinning file and are updated by `conda smithy rerender` when the pins are updated. It isn't quite time to do that as far as I know.

What caused you to need to bump the GDAL version? Was there some incompatibility?

**Comment by SrNetoChan** [comment]
> What caused you to need to bump the GDAL version? Was there some incompatibility?

@hobu wanted to add PDAL to the QGIS feedstock to have point cloud support for QGIS 3.18

File: .ci_support/linux_64_.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ hdf5:
 jsoncpp:
 - 1.9.4
 libgdal:
-- '3.1'
+- '3.2'
```
Comment (ocefpaf) [inline .ci_support/linux_64_.yaml:28]: We are fixing a bug that should do this automatically soon. Forcing it like that won't persist b/c a re-render will revert to what is in the pinning file.

**Comment by ocefpaf** [comment]
Let's wait for the fix in the bot to issue the libgdal 3.2 PR. It should be out later today. See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1271 and https://github.com/regro/cf-scripts/issues/1346 for context.

**Comment by SrNetoChan** [comment]
@ocefpaf that explains why QGIS is using a different version of GDAL. Let's close this one.

**Pull request closed without merging**